src/higley_lab_to_nwb/benisty_2024/benisty_2024_convert_dual_imaging_session.py

```diff
@@ -58,13 +59,14 @@ def dual_imaging_session_to_nwb(
     csv_file_paths = glob.glob(os.path.join(folder_path, f"{session_id}*.csv"))
     if csv_file_paths:
         csv_file_path = csv_file_paths[0]
-        mat_file_path = os.path.splitext(csv_file_path)[0] + '.mat'
+        mat_file_path = os.path.splitext(csv_file_path)[0] + ".mat"
```

Comment:
can one minor suggestion, I would encourage the usage of `pathlib`
for instance, instead of glob.glob
you can do:
```
from pathlib import Path
folder_path = Path(folder_path)
csv_file_paths = list(folder_path.glob(f"{session_id}*.csv"))
...
mat_file_path = f"{csv_file_paths[0].stem}.mat"
```
no need for import os or glob